### ISSUE TITLE
Adapters for models through API

### ISSUE THREAD

**Issue by vishal-ramesh2** [adapter-hub/adapter-transformers#491]
I have a pre-trained BERT model that can only be accessed through a REST API. How can I use adapters with that model? I need to fine-tune the model on different datasets. How can I achieve this using Adapter Transformers?

**Comment by adapter-hub-bert** [comment]
This issue has been automatically marked as stale because it has been without activity for 90 days. This issue will be closed in 14 days unless you comment or remove the stale label.

**Comment by adapter-hub-bert** [comment]
This issue was closed because it was stale for 14 days without any activity.
